utils/utils.go

```diff
@@ -28,3 +30,27 @@ func WrapCloseErr(err error, closer io.Closer) error {
 func EvmToCosmosAddress(addr common.Address) sdktypes.AccAddress {
 	return addr.Bytes()
 }
+
+// DeriveL1BlockInfoToProto converts a derive.L1BlockInfo to our L1BlockInfo proto message
+func DeriveL1BlockInfoToProto(info *derive.L1BlockInfo) *types.L1BlockInfo {
+	protoL1BlockInfo := &types.L1BlockInfo{
+		Number:            info.Number,
+		Time:              info.Time,
+		BlockHash:         info.BlockHash[:],
+		SequenceNumber:    info.SequenceNumber,
+		BatcherAddr:       info.BatcherAddr[:],
+		L1FeeOverhead:     info.L1FeeOverhead[:],
+		L1FeeScalar:       info.L1FeeScalar[:],
+		BaseFeeScalar:     info.BaseFeeScalar,
+		BlobBaseFeeScalar: info.BlobBaseFeeScalar,
+	}
+
+	if info.BaseFee != nil {
+		protoL1BlockInfo.BaseFee = info.BaseFee.Bytes()
+	}
+	if info.BlobBaseFee != nil {
+		protoL1BlockInfo.BlobBaseFee = info.BlobBaseFee.Bytes()
+	}
```

Comment:
@natebeauregard they're both pointer types...
the BlobBaseFee is null when running the test and causes a null pointer dereference. i did the check to avoid it

Reply:
Ah yes! I glossed over pulling the bytes from the field. I think your approach makes sense